### DISCUSSION TITLE
Homebridge how-to?

### DISCUSSION THREAD

**Discussion by dmindfuse** [john30/ebusd#579] [Q&A] [answered]
Hi everyone. I'm keen to integrate my heating with Apple Homekit using ebus adapter 3. I came across various solutions to integrate third party devices into Homekit, of which I remember: Home Assistant, IoBroker, Node Red, Homebridge. By far the simplest to use for me from a time investment perspective appears to be Homebridge. So I set out to look how I can integrate ebus data into Homebridge, but found absolutely nothing about. Am I on the wrong track here and is homebridge a bad idea for ebus3? In which case... how do Home Assistant and Iobroker fit the prospect of Homekit integration?
Cheers
MF

**Comment by john30** [comment, marked as answer]
depends on how homebridge can handle MQTT messages. ebusd publishes almost everything to MQTT, so you'd have to check the possibilities on homebridge side

**Reply by dmindfuse** [reply]
Thanks, I see now. It appears there is a MQTT-thing plugin for Homebridge. Needs further exploration. 🫡

**Reply by wimleers** [reply]
@dmindfuse I'm exploring this too! (But have yet to receive an `ebus` adapter.) What are your conclusions so far?

**Reply by dmindfuse** [reply]
Hoi Wim. I didn’t pursue the effort much. Instead I replaced my router with a J4125 mini pc and ESXi 😂. The underlying idea I had was maybe to integrate a remote thermostat in every room, but when I realized how much more effort it was going to be plus the risk of screwing up my heating I discarded the idea. Sorry. Thought I respond anyhow… have a good one.

**Reply by HolyGiraffe** [reply]
@wimleers 
Hi! I am currently setting up a mqtt server between ebusd <-> homebridge (<-Plugin "Homebridge Mqttthing")
I'll update if this works and how it appears in the home app

**Reply by wimleers** [reply]
@HolyGiraffe I finally have an eBUS adapter too! What was your Homebridge MQTT experience like?
